scripts/run-postgres-tests.sh

```
@@ -0,0 +1,64 @@
+
+echo "=== Initializing Postgres Test Database ==="
+PGHOST="$PG_HOST" \
+PGPORT="$PG_PORT" \
+PGUSER="$PG_ADMIN_USER" \
+PGPASSWORD="$PGPASSWORD" \
+PGDATABASE="$PC_DB_NAME" \
+PGOPTIONS="--search_path=public" \
+pkg/db/migrations/init/postgres.sh "$PC_DB_NAME" "$PC_DB_USER" "$PC_DB_PASSWORD"
+
+echo "=== Migrating Postgres Test Database ==="
+PC_POSTGRES="postgres://${PG_ADMIN_USER}:${PGPASSWORD}@${PG_HOST}:${PG_PORT}/${PC_DB_NAME}?search_path=public" \
```

Shellcheck (0.11.0)

[warning] PC_POSTGRES appears unused. Verify use (or export if used externally).

(SC2034)